layouts/diagnostic.html

```diff
@@ -0,0 +1,13 @@
+<h1 id="diagnostic">Diagnostic</h1>
+
+<p>The Diagnostic resources are concerned with the process of observing, testing, and diagnosing a patient. Note, once a
+  diagnosis is confirmed, it would be recorded in Condition so that it is part of the historical patient record.</p>
+
+<p>The Observation resource is primarily concerned with recording discrete values, answers, and results such as clinical
```

Comment:
Is there a benefit to including this paragraph when Observation is not available?